.circleci/config.yml

```diff
@@ -138,37 +138,47 @@ jobs:
           paths:
             - front/dist
 
+  lint-front:
```

Comment:
revert the circleci config so that the tests with react-testing library still run.